lib/policy_machine.rb

```diff
@@ -90,16 +90,18 @@ def add_association(user_attribute_pe, operation_set, object_attribute_pe)
   # TODO: add option to ignore policy classes to allow consumer to speed up this method.
   # TODO: Parallelize the two component checks
   def is_privilege?(user_or_attribute, operation, object_or_attribute, options = {})
-    is_privilege_ignoring_prohibitions?(user_or_attribute, operation, object_or_attribute, options) &&
-      (options[:ignore_prohibitions] || !is_privilege_ignoring_prohibitions?(user_or_attribute, PM::Prohibition.on(operation), object_or_attribute, options))
+    options_copy = options.deep_dup
```

Comment:
WHy is this necessary?

Reply:
For some reason, the options hash values were wiping themselves after running `!is_privilege_ignoring_prohibitions?(user_or_attribute, PM::Prohibition.on(operation), object_or_attribute, options)` This was a quickfix to see if any other specs were broken. Should I investigate the cause or this is a viable solution?

Reply:
YIkes. Okay, you don't need to investigate this, but please add a comment.

I would _prefer_ that you investigated and fixed the consuming method not to mutate its input, though :)

Reply:
I'll take a look!